### DISCUSSION TITLE
Option to stretch aspect ratio.

### DISCUSSION THREAD

**Discussion by UrsoManso** [ikemen-engine/Ikemen-GO#1745] [Feature Request]
### Is your feature request related to a problem? Please describe.

The vast majority of the MUGEN content is in 4:3, and having to manually edit every single asset for widescreen localcoord is too much unnecessary work unless you really want to play in proper widescreen.

### Describe the solution you'd like

Winmugen allows you to stretch 4:3 localcoord to widescreen window resolutions. Would be nice to have this feature that many emulators have on Ikemen to save time for those who want to play on widescreen but can't be bothered to edit everything.

### Additional context

4:3
![137447502-aa61c86c-c470-4595-84d3-be378e03cf3d 1](https://github.com/ikemen-engine/Ikemen-GO/assets/77592586/2bb9a7fd-d0ee-40e0-995f-934e8fac4536)

Stretched to 16:9
![137447504-86adc1c5-0ab0-4679-9648-86600692b117 1](https://github.com/ikemen-engine/Ikemen-GO/assets/77592586/f3e0fe27-2d6e-414f-9d0f-b6ecb2b5fefa)

**Comment by potsmugen** [comment]
We could (should?) have game resolution be a separate parameter from video resolution.

**Reply by UrsoManso** [reply]
I play on a 1080p monitor, when I set the resolution to 480p and launch fullscreen the game is perfectly stretched to 16:9. But this only works on fullscreen and 480p alone. Being able to do something like that on windowed mode would be golden.

**Comment by SuperFromND** [comment]
I've recently made a bunch of changes to allow for resizable windows that now allows you to do this (among other things) in the latest nightly builds. Set `KeepAspect` in `save/config.json` to `false` to stretch the game to fill the window, or set it to `true` to preserve the aspect ratio:

https://github.com/user-attachments/assets/67b9d607-57ed-430b-96fa-e143568e3b8c
